ShareBook/ShareBook.Api/Controllers/BookController.cs

```diff
@@ -229,6 +232,30 @@ public IActionResult RequestBook([FromBody] RequestBookVM requestBookVM)
             return Ok(new Result { SuccessMessage = "Pedido realizado com sucesso!" });
         }
 
+        [HttpPost("CancelRequest/{requestId}")]
+        [Authorize("Bearer")]
+        public IActionResult CancelRequest(Guid requestId)
+        {
+            var request = _bookUserService.GetRequest(requestId);
+
+            if (request != null)

```

Comment:
Não impeditivo. Quero ver se vc concorda. O que acha de usar early return? Concorda que o código ficaria mais limpo e escalável?

exemplo
![image](https://user-images.githubusercontent.com/10622114/179001585-4e1ff9de-4fac-4334-8249-60ce77cc9c1c.png)

fonte: https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement

Reply:
Faz sentido! E fica mais legível. Vou adicionar um commit